### PR TITLE
Add layer attributions to print

### PR DESCRIPTION
**Pull request by LukasLohoff**

### CODE REVIEW

**Comment by mholthausen** [comment]
Always adds the attributions of the layers to the parameters so that they are included in a printout

@terrestris/devs please review

File: src/components/PrintForm/index.tsx [hunk 1]
```diff
@@ -186,12 +238,16 @@ export const PrintForm: React.FC<PrintFormProps> = ({
       pManager.setDpi(pManager.getDpis()[0]);
       pManager.setLayout(pManager.getLayouts()[0]?.name);
 
+      dispatch(addCustomParam({
+        ['attributions']: getAttributions(pManager)
```
Comment (dnlkoch) [inline src/components/PrintForm/index.tsx:242]: Why do we need to pass the key `attributions` as array?

File: src/components/PrintForm/index.tsx [hunk 2]
```diff
@@ -120,6 +132,46 @@ export const PrintForm: React.FC<PrintFormProps> = ({
...
+      return layerName &&
+        !(layerName.includes('react-geo')) &&
+        layer.getVisible() &&
+        !(layer instanceof OlLayerGroup) &&
+        layer !== printLayer;
+    }) as LayerType[];
+  }, [map]);
+
+  const getAttributions = useCallback((manager: MapFishPrintV3Manager) => {
+    if (!manager) {
```
Comment (dnlkoch) [inline src/components/PrintForm/index.tsx:151]: According to the typing this check shouldn't be needed, if this is actually optional, we should mark it accordingly.

**Review by dnlkoch — APPROVED** [review]

File: src/components/PrintForm/index.tsx (hunk 2 above)
Comment (mholthausen) [reply src/components/PrintForm/index.tsx]: The unnecessary check got removed.

**Pull request merged**